releases/nightly-build/compose-files/docker-compose-nexus-no-secty.yml

```diff
@@ -27,8 +27,10 @@ services:
   app-service-rules:
     container_name: edgex-app-service-configurable-rules
     depends_on:
-    - consul
-    - data
+      consul:
```

Comment:
These are new. I assume you didn't manually change these, correct? 
What version of docker compose do you have installed?

Reply:
@zhanghongtong , I have confirmed that there is a bug in the latest version of docker-compose (1.27.3) that is causing this incorrect generation of the `depends_on` section. Please downgrade to docker-compose version 1.27.2 and regenerate these compose files.

You can download the docker-compose 1.27.2 artifacts from here:
https://github.com/docker/compose/releases/tag/1.27.2